both/accounts/config.js

```diff
@@ -1,3 +1,31 @@
+var pwd = AccountsTemplates.removeField('password');
```

Comment:
Please rename this variable to be more meaningful. You can use 'camelCase' notation to allow for multiple words in the variable name.